src/adapter/src/coord.rs

```diff
@@ -2094,6 +2154,81 @@ impl<S: Append + 'static> Coordinator<S> {
                 });
             }
 
+            stmt @ Statement::CreateConnection(_) => {
+                use mz_storage::client::connections::Connection::*;
```

Comment:
Light preference for something (?) to be cut to its own function here. Maybe two matched statements each get their own fn, similar to how handle_statement is called.